<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>空文件夹扫描</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="1">
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">work-break: break-all;</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;每行填写一条规则&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;符号 * 代表匹配 0-1 个文件夹,&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;例如 D:/example/* 将匹配所有D:/example下的文件夹，但是不含匹配文件夹中的文件夹。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>路径模式</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>正则筛选</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEditRegexPattern"/>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>使用正则表达式筛选匹配结果</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QTextEdit" name="textEditPatterns">
               <property name="placeholderText">
                <string>请输入路径匹配模式</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButtonPath">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>空文件夹</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonRefresh">
             <property name="text">
              <string>刷新</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListView" name="listView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QToolButton" name="toolButtonTxt">
             <property name="text">
              <string>生成txt</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignRight">
            <widget class="QLabel" name="labelVersion">
             <property name="text">
              <string>version</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>446</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="actionAskPath">
   <property name="text">
    <string>AskPath</string>
   </property>
  </action>
  <action name="actionTxt">
   <property name="text">
    <string>txt</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButtonTxt</sender>
   <signal>clicked()</signal>
   <receiver>actionTxt</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonRefresh</sender>
   <signal>clicked()</signal>
   <receiver>actionRefresh</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonPath</sender>
   <signal>clicked()</signal>
   <receiver>actionAskPath</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
